presto-spi/src/main/java/com/facebook/presto/spi/security/SystemAccessControl.java

```diff
@@ -302,22 +301,4 @@ default void checkCanRevokeTablePrivilege(Identity identity, Privilege privilege
     {
         denyRevokeTablePrivilege(privilege.toString(), table.toString());
     }
-
-    /**
```

Comment:
same for this one.

Reply:
Restored in `Add access control checks for SHOW ROLES`